Tasks/day.js

```diff
@@ -1,15 +1,14 @@
 // Get day number
+'use strict'
```

Comment:
```suggestion
'use strict';
```
Use ; after 'use strict'